app/assets/javascripts/actions/story.js

```diff
@@ -88,6 +88,19 @@ export const setLoadingStory = (id, from) => ({
   from
 });
 
+export const confirmBeforeSaveIfNeeded = async (story, confirm, needConfirmation, callback) => {
+  const confirmStoryChange = story => confirm(I18n.t('story.definitive_sure', { action: `${story.state}` }))
```

Comment:
This is making it be shown as:

"Are you sure you want to _accepted_ this story"

It should be _accept_, can you check?